bigchaindb/backend/mongodb/query.py

```diff
@@ -70,17 +81,20 @@ def get_transaction_from_block(conn, transaction_id, block_id):
 
 @register_query(MongoDBConnection)
 def get_transaction_from_backlog(conn, transaction_id):
-    return conn.db['backlog']\
-               .find_one({'id': transaction_id},
-                         projection={'_id': False, 'assignee': False,
-                                     'assignment_timestamp': False})
+    return conn.run(
+        conn.collection('backlog')
+        .find_one({'id': transaction_id},
+                  projection={'_id': False,
+                              'assignee': False,
+                              'assignment_timestamp': False}))
 
 
 @register_query(MongoDBConnection)
 def get_blocks_status_from_transaction(conn, transaction_id):
-    return conn.db['bigchain']\
-            .find({'block.transactions.id': transaction_id},
-                  projection=['id', 'block.voters'])
+    return conn.run(
+        conn.collection('bigchain')
+        .find({'block.transactions.id': transaction_id},
+              projection=['id', 'block.voters']))
 
 
```

Comment:
I think `get_txids_filtered` was added recently and it also needs to use connection.run